### PR TITLE
Badge button

### PR DESCRIPTION
**Pull request by deekshithmd**
created badge and button components review comments invited for further improvement

### CODE REVIEW

**Comment by pankaj-prog** [comment]
Changes you can make : 
1. Make avatar inline block elements
2. In button component -  last button arrow is not visible. Check for the contrast while choosing the  colors.

**Comment by deekshithmd** [comment]
> Changes you can make :
> 
> 1. Make avatar inline block elements
> 2. In button component -  last button arrow is not visible. Check for the contrast while choosing the  colors.

ok, thank you I will work on that

File: index.html [hunk 1]
```diff
@@ -4,50 +4,118 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
     <link rel="stylesheet" href="style.css">
     <title>Components</title>
 </head>
 <body>
-    <h1>Avatar Components</h1>
```
Comment (aiman-ansari) [inline index.html:11]: It could be better if you wrap your html code inside a div. 
Otherwise everything is good to commit.

**Review by aiman-ansari — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (deekshithmd) [reply index.html:11]: yes for documentation I will do that and for individual items I felt it is not required, Thank you for the review

**Pull request merged**